docs/source/index.rst

```diff
@@ -114,18 +114,18 @@ Please see our `contributing guidelines <https://github.com/cleanlab/cleanlab/bl
    :hidden:
    :caption: Tutorials
 
-   tutorials/indepth_overview
-   tutorials/image
-   tutorials/text
-   tutorials/tabular
-   tutorials/audio
-   tutorials/dataset_health
-   tutorials/outliers
-   tutorials/multiannotator
-   tutorials/multilabel_classification
-   tutorials/token_classification
-   tutorials/pred_probs_cross_val
-   tutorials/faq
+   Getting Started With Cleanlab <tutorials/indepth_overview>
+   Image Classification (Pytorch) <tutorials/image>
+   Text Classification (Tensorflow) <tutorials/text>
+   Tabular Classification     (Scikit-Learn) <tutorials/tabular>
```

Comment:
```suggestion
   Tabular Classification (sklearn) <tutorials/tabular>
```

Reply:
To be clear, only suggesting ^ if it makes the short title fit on a single line, otherwise let's keep the current one. Also if the current one here already fits on a single line as well, then let's keep it, but I'm assuming it's taking up 2 lines right now

Reply:
This and audio is two lines! Wasn't sure if referring to Scikit learn as sklearn was polite.